src/Credentials.ts

```diff
@@ -163,13 +169,16 @@ interface VerificationParam {
   exp?: number
   vc?: string[]
   callbackUrl?: string
+  nbf?: number,
+  expiresIn?: number
```

Comment:
is this meant to align the API for different requests?

Reply:
correct, since the new did-jwt requires expiresIn